saleor/core/utils/url.py

```diff
@@ -33,3 +36,17 @@ def prepare_url(params: str, redirect_url: str) -> str:
     split_url = urlsplit(redirect_url)
     split_url = split_url._replace(query=params)
     return split_url.geturl()
+
+
+class APPURLValidator(URLValidator):
+    validator = URLValidator
+    host_re = "(" + validator.hostname_re + validator.domain_re + "|localhost)"
+    regex = _lazy_re_compile(
+        r"^(?:[a-z0-9.+-]*)://"  # scheme is validated separately
```

Comment:
Typo?
```suggestion
        r"^(?:[a-z0-9.+-]*)://"  # schema is validated separately
```

Reply:
`Scheme` its for URLs
https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#scheme

Reply:
👍